src/url.js

```diff
@@ -376,6 +396,23 @@ export function isProtocolValid(url) {
   return !INVALID_PROTOCOLS.includes(url.protocol);
 }
 
+/**
+ * Returns a URL without AMP JS parameters.
+ * @param {string} url
+ * @return {string}
+ */
+export function removeAmpJsParamsFromUrl(url) {
```

Comment:
can you rename the other method `removeAmpJsParams`, maybe to `removeAmpJsParamsFromSearch`.